nixos/modules/services/misc/ersatztv.nix

```diff
@@ -0,0 +1,126 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkIf
+    getExe
+    maintainers
+    mkEnableOption
+    mkOption
+    mkPackageOption
+    ;
+  inherit (lib.types) str path bool;
+  cfg = config.services.ersatztv;
+in
+{
+  options = {
+    services.ersatztv = {
+      enable = mkEnableOption "ErsatzTV";
+
+      package = mkPackageOption pkgs "ersatztv" { };
+
+      user = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "User account under which ErsatzTV runs.";
+      };
+
+      group = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "Group under which ErsatzTV runs.";
+      };
+
+      transcodeDir = mkOption {
+        type = path;
+        default = "/var/lib/etv-transcode";
+        description = ''
+          Temporary transcode directory
+        '';
+      };
```

Comment:
I'd make both `transcodeDir` and `configDir` sub-folders of the service's state directory, rather than two different folders under `/var/lib`.

Something like: `/var/lib/ersatztv/transcode` and `/var/lib/ersatztv/config`.